programs/cleos/main.cpp

```diff
@@ -445,13 +445,13 @@ auto abi_serializer_resolver = [](const name& account) -> fc::optional<abi_seria
    static unordered_map<account_name, fc::optional<abi_serializer> > abi_cache;
    auto it = abi_cache.find( account );
    if ( it == abi_cache.end() ) {
-      auto result = call(get_abi_func, fc::mutable_variant_object("account_name", account));
-      auto abi_results = result.as<eosio::chain_apis::read_only::get_abi_results>();
-
-      fc::optional<abi_serializer> abis;
-      if( abi_results.abi.valid() ) {
-         abis.emplace( *abi_results.abi, abi_serializer_max_time );
-      } else {
+       fc::optional<abi_serializer> abis;
+       auto raw_abi_result = call(get_raw_abi_func, fc::mutable_variant_object("account_name", account));
+       auto raw_abi_blob = raw_abi_result["abi"].as_blob().data;
+       if (raw_abi_blob.size() != 0) {
+           auto abi_result = fc::raw::unpack<abi_def>(raw_abi_blob);
+           abis.emplace(abi_result, abi_serializer_max_time);
+       } else {
```

Comment:
It's hard to tell on Github, but it looks like you changed the indentation here. In general, most of our code uses 3 spaces.

Reply:
Make sure you are not using tabs.